src/libData/AccountData/AccountStoreSC.tpp

```diff
@@ -211,12 +211,18 @@ uint64_t AccountStoreSC<MAP>::InvokeEvmInterpreter(
       // Get the account that this apply instruction applies to
       Account* targetAccount = this->GetAccount(Address(it->Address()));
       if (targetAccount == nullptr) {
-        LOG_GENERAL(
-            WARNING,
-            "Cannot find account for address given in  Apply operation from "
-            "EVM-DS"
-            " ");
-        return gas;
+        if (!this->AddAccount(Address(it->Address()), {0, 0})) {
```

Comment:
is this creating an empty account if there is an apply instruction? In my mind, the apply instruction would usually contain code/data

Reply:
The code gets added later which could be a small optimisation, to avoid creating with zero and then setting later, it allows for the code to be modified on an existing account. It also allows an account to be created with balance and no code if the code is omitted by EVM-ds